packages/react/test/specs/components/TextArea/TextArea-test.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react'
+import { ReactWrapper } from 'enzyme'
+import { mountWithProvider as mount } from 'test/utils'
+import TextArea from 'src/components/TextArea/TextArea'
+import { isConformant } from 'test/specs/commonTests'
+
+const getTextAreaDomNode = (textAreaComp: ReactWrapper): HTMLTextAreaElement =>
+  textAreaComp.find('textarea').getDOMNode() as HTMLTextAreaElement
+
+const setUserTextAreaValue = (textAreaComp: ReactWrapper, value: string) => {
+  textAreaComp.find('textarea').simulate('change', { target: { value } })
+}
+
+const testValue = 'test value'
+
+describe('TextArea', () => {
+  describe('conformance', () => {
+    isConformant(TextArea, {
+      eventTargets: {
+        onChange: 'textarea',
+        onKeyDown: 'textarea',
+        onKeyPress: 'textarea',
+        onKeyUp: 'textarea',
```

Comment:
Not sure, but `textarea` is `root` so `isConformant(TextArea)` should work without any config